test/Engine/ProtoTest/Associative/SimpleVerification.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+using ProtoCore.AST.AssociativeAST;
+using ProtoCore.DSASM.Mirror;
+using ProtoCore.Lang;
+using ProtoTest.TD;
+using ProtoTestFx.TD;
+namespace ProtoTest.Associative
+{
+    public class SimpleVerification : ProtoTestBase
+    {
+        [Test]
+        public void TestAssignment01()
+        {
+
+            String code =
+@"
a = 1;
";
+            thisTest.RunAndVerify(
+                code, 
+                TestFrameWork.BuildVerifyPair("a", 1)
+                );
+        }
+
+        [Test]
+        public void TestAssignment02()
+        {
+            string code =
+@"
a = 1;
b = 2;
";
+            thisTest.RunAndVerify(
+                code,
+                TestFrameWork.BuildVerifyPair("a", 1),
+                TestFrameWork.BuildVerifyPair("b", 2)
+                );
+        }
+
+
+        [Test]
+        public void TestFunctionCall01()

```

Comment:
Yea we can go directly to the string format. Ill define the string formatting in the doc itself:
https://docs.google.com/document/d/1pG6DXp3iFWbavTElm1KZtWRRiBzkFXVl_ZDLzEnacHw/edit